src/KnowbaseItem.php

```diff
@@ -1615,12 +1615,8 @@ public static function showList($options, $type = 'search')
                 }
 
                 if (isset($options['item_itemtype'], $options['item_items_id']) && ($output_type === Search::HTML_OUTPUT)) {
-                    $forcetab = $options['item_itemtype'] . '$main';
-                    $item_itemtype = $options['item_itemtype'];
-                    $content = "<a href='" . $item_itemtype::getFormURLWithID($options['item_items_id']) .
-                        "&amp;load_kb_sol=" . $data['id'] .
```

Comment:
Il the `load_kb_sol` related code still used somewhere else? It it is not used anymore, you could clean the ajax, front and class code.

Reply:
It should 
I didn't the current diff, but it corresponds to the action of moving back grom the kb to the ticket with content from an article to the solution

Reply:
Ok I checked out and tested the pr.
The parameter is indeed probably not used anymore.